hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthStatus.java

```diff
@@ -87,19 +117,19 @@ public boolean isDeleted() {
   }
 
   public boolean isOverReplicated() {
-    return replicaDelta < 0;
+    return containerReplicaCount.isOverReplicated();
   }
 
   public boolean isUnderReplicated() {
-    return !isMissing() && replicaDelta > 0;
+    return !isMissing() && containerReplicaCount.isUnderReplicated();
```

Comment:
@sodonnel By removing `isUnderReplicated()` from the interface, this should be
```suggestion
    return !isMissing() && !containerReplicaCount.isSufficientlyReplicated();
```
right?